### PR TITLE
docs: Adjust a11y documentation

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/features/working-with-accessibility.md [hunk 1]
```diff
@@ -90,13 +90,10 @@ When the value of `WinRTFeatureConfiguration.Accessibility.HighContrast` is chan
 
 ## Known issues
 
-- `Hyperlink` in `TextBlock` is not supported.
-- `TextBox` and `PasswordBox` don't use `Header` and `PlaceholderText`.
 - `ItemsControl` doesn't use `AutomationProperties.Name` on its `DataTemplate`'s root.
 - There are XAML code generation conflicts between `x:Name` and `AutomationProperties.Name`.
 - A child with the same accessible name as its parent is accessibility focusable.
 - `Control` doesn't receive focus when accessibility focused.
-- `TabIndex` is not supported.
```
Comment (Youssef1313) [inline doc/articles/features/working-with-accessibility.md:99]: Are you sure this is supported from a11y point of view?
i.e, if you use TalkBack will it respect TabIndex? I thought TabIndex implementation currently works with focus manager only for tab navigation?

File: doc/articles/features/working-with-accessibility.md [hunk 2]
```diff
@@ -90,13 +90,10 @@ When the value of `WinRTFeatureConfiguration.Accessibility.HighContrast` is chan
 
 ## Known issues
 
-- `Hyperlink` in `TextBlock` is not supported.
```
Comment (Youssef1313) [inline doc/articles/features/working-with-accessibility.md:93]: Same here, is this supported from a11y point of view?
I think screen readers are expected to read the text, be able to tell it's hyperlink, and double tapping with a11y focus should open the link. Does it work that way?

File: doc/articles/features/working-with-accessibility.md [hunk 3]
```diff
@@ -90,13 +90,10 @@ When the value of `WinRTFeatureConfiguration.Accessibility.HighContrast` is chan
...
-- `TextBox` and `PasswordBox` don't use `Header` and `PlaceholderText`.
```
Comment (Youssef1313) [inline doc/articles/features/working-with-accessibility.md:94]: Same here, I think on a11y focus it's expected that screen readers will read the header and placeholder.

File: doc/articles/features/working-with-accessibility.md (hunk 1 above)
Comment (MartinZikmund) [reply doc/articles/features/working-with-accessibility.md:99]: Good point, I read these only from the implementation perspective, not A11y perspective. Makes sense to keep it there then

**Pull request closed without merging**